src/number_theory/number_field.lean

```diff
@@ -43,6 +43,21 @@ class number_field (K : Type*) [field K] : Prop :=
 open function
 open_locale classical big_operators
 
+/-- `ℤ` with its usual ring structure is not a field. -/
+lemma int.not_is_field : ¬ is_field ℤ :=
+begin
+  rw ring.not_is_field_iff_exists_ideal_bot_lt_and_lt_top,
```

Comment:
I didn't try it, so I can very well miss something, but this proof seems a little overpowered. Isn't simpler to say that `1` is odd so not even so `2` hasn't an inverse?

Reply:
Good point, this was overkill. I changed the proof.